client.go

```diff
@@ -590,4 +591,36 @@ func (c *Client) CreateMonitoredItems(subID uint32, ts ua.TimestampsToReturn, it
 	}
 
 	return respItems, nil
+
+}
+
+func (c *Client) HistoryReadRawModified(nodes []*ua.HistoryReadValueID, isReadModified bool) (*ua.HistoryReadResponse, error) {
```

Comment:
The version below probably more flexible. I'm no expert on history reads. Is HistoryReadRawModified the only history read we can do or are there others? Can we name it `HistoryRead` or is that ambiguous?

```go
func (c *Client) HistoryRead(nodes []*ua.HistoryReadValueID, details *ua.ReadRawModifiedDetails) {...}
```

Reply:
There is 6 different type of `HistoryReadDetails`. I changed the code so we can use it for 2 of them (Part 11 Table 19).